### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by TheRealHaui**
Added Unit Tests to increase code coverage.

### CODE REVIEW

File: jetty-start/src/test/java/org/eclipse/jetty/start/UtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    }
+
+    @Test
+    public void testJoin() {
+        assertEquals("", Utils.join((Collection<?>) null, "V9ewe2K"));
+    }
+
+    @Test
+    public void testJoinTaking4ArgumentsAndReturningEmptyString() {
+        assertEquals("", Utils.join((Object[]) null, (-3563), 1051, ""));
```
Comment (joakime) [inline jetty-start/src/test/java/org/eclipse/jetty/start/UtilsTest.java]: Interesting interaction here.

File: jetty-start/src/test/java/org/eclipse/jetty/start/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    }
+
+    @Test
+    public void testIsBlankUsingNull() {
+        assertTrue(Utils.isBlank(null));
+    }
+
+    @Test
+    public void testJoin() {
+        assertEquals("", Utils.join((Collection<?>) null, "V9ewe2K"));
```
Comment (joakime) [inline jetty-start/src/test/java/org/eclipse/jetty/start/UtilsTest.java]: This is a curious interaction as well.
I wonder if we should just use `java.util.StringJoiner` instead and deprecate this `Utils.join` method entirely?

File: jetty-start/src/test/java/org/eclipse/jetty/start/UtilsTest.java [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+public class UtilsTest{
+
+    @Test
+    public void testIsNotBlankReturningTrue() {
+        assertTrue(Utils.isNotBlank(" @t;3/|O)t"));
+    }
+
+    @Test
+    public void testIsNotBlankReturningFalse() {
+        assertFalse(Utils.isNotBlank(null));
```
Comment (joakime) [inline jetty-start/src/test/java/org/eclipse/jetty/start/UtilsTest.java]: This one should be marked as `@Deprecated` and removed in jetty-10.0.x, as `String.isBlank()` exists in Java 11+.

**Comment by joakime** [comment]
Did you encounter some issues in the `jetty-start` `Utils` class that made you want to create these unit tests?

**Comment by TheRealHaui** [comment]
@joakime 
I had some issues with git and my IDE playing tricks on me regarding the sign-off message.
Therefore I had to delete my commits entirely in order to pass the Eclipse EULA checks.
Didn't have anything to do with your comments.
Now I see due to my push --force the pull request was closed automatically.
Well, ...

Shall I create a new pull request?

Regarding your question:
No, I did not encounter issues.
I just wanted to increase test code coverage in order to detect regressions in the case of changes automatically.
Shall I go on with this?

**Comment by joakime** [comment]
Please, better test coverage is always a good thing.

**Comment by TheRealHaui** [comment]
Great, I appreciate that because I enjoy writing automated tests.
However, many do not - for what reason ever ... - welcome this.

So, after "discussing" the sign-off message thing with git/my IDE again I am going to create a new pull request with the tests.
Thanks, for your response!

**Pull request closed without merging**